### PR TITLE
[freetds] Remove patches and update to a version that no longer needs them

### PR DESCRIPTION
**Pull request by BillyONeal**
Upstream seems to have version numbered releases but they aren't explicitly numbered well in GitHub, so I've intentionally changed the version-string to something unorderable for now.

### CODE REVIEW

**Comment by NancyLi1013** [comment]
The hash value seems not correct.

```
Starting package 2/2: freetds:x86-windows
Building package freetds[core,openssl]:x86-windows...
Could not locate cached archive: W:\eb\eb83eef515d3c211be44328d6f11fc898e8bdc97.zip
-- Downloading https://github.com/freetds/freetds/archive/d33674659aa7d70cf713c8920850c4d6751dbd95.tar.gz...
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:103 (message):
  

  File does not have expected hash:

          File path: [ D:/downloads/temp/freetds-freetds-d33674659aa7d70cf713c8920850c4d6751dbd95.tar.gz ]
      Expected hash: [ a01f3bb0b61459e97e50e0ed6ac0f7b2f17efa18 ]
        Actual hash: [ 1df12fea36f38a8ec3f6509ce72781a82304c8444358e4eb907a1a01a5a254419b2e7ce7d24ae94e961f0725317f6e57f664a7c474c1a89e0f0de71124b0f88d ]

  The file may have been corrupted in transit.  This can be caused by
  proxies.  If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY
  environment variables to
  "https://user:***@your-proxy-ip-address:port/".
```

File: ports/freetds/portfile.cmake [hunk 1]
```diff
@@ -1,12 +1,9 @@
 vcpkg_from_github(
```
Comment (NancyLi1013) [inline ports/freetds/portfile.cmake:1]:
```suggestion
vcpkg_fail_port_install(ON_TARGET "uwp" "linux" "osx")

vcpkg_from_github(
```
Maybe We can consider to add this unsupported messages for this port.

**Comment by freddy77** [comment]
Hi, please use commit 16f7a6280c7a19bfe5c60e5d61cc08e3f2dff991 instead, see https://github.com/FreeTDS/freetds/commit/16f7a6280c7a19bfe5c60e5d61cc08e3f2dff991. This is version 1.2.11. The commit you used is from master branch which is supposed to be development branch, not production one. You can also download the packaged file from https://www.freetds.org (like ftp://ftp.freetds.org/pub/freetds/stable/freetds-1.2.11.tar.gz).

**Comment by NancyLi1013** [comment]
@freddy77 

Thanks for your kindly reminder.

**Comment by BillyONeal** [comment]
@freddy77 Done, and thanks!

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
All checks have passed now.

**Comment by strega-nil** [comment]
Thanks @BillyONeal !

**Pull request merged**